.env.example

```
@@ -1,3 +1,7 @@
 WEATHER_API_KEY=
 NEXT_PUBLIC_KAKAO_MAP_KEY=
-BASE_URL=
\ No newline at end of file
+BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASE_URL key should go before the NEXT_PUBLIC_KAKAO_MAP_KEY key

(UnorderedKey)